dist/s_goto.py

```diff
@@ -0,0 +1,18 @@
+# Trim all trailing whitespace in front of goto labels.
+# This is a workaround for a Clang Format limitation where goto labels are
+# automatically indented according to nesting.
+import re, sys
+
+# 1. Zero or more whitespace characters.
+# 2. One or more lowercase ASCII characters.
+# 3. Colon character.
+p = re.compile('^\s*[a-z]+:$')
```

Comment:
There are some goto statements in the code as "err: /*", the clang format is not bringing the comments into the next line like statements, so the newly added goto script is missing to adjust these gotos. IMO, it is better to move the comment into the new line, so that this s_goto can work properly.

Reply:
Thanks for picking up on that. I agree that the comment should just go underneath the tag.
Before applying this change into `develop`, I'm going to do a PR to some preliminary cleanup like remove some low value macros. I'll also include this change at the same time.